### ISSUE TITLE
Smitten Kitchen (Wordpress) Scraper Request

### ISSUE THREAD

**Issue by StarkZarn** [hhursev/recipe-scrapers#516]
We always enjoy adding support for new websites to the library!

To help us out, please check that recipes published on the website you're requesting are public (we can't currently scrape recipes that require an account login) and add sample recipe URL(s) below:

- https://smittenkitchen.com/2015/03/baked-chickpeas-with-pita-chips-and-yogurt/

Can you write Python and would you like to help add the scraper yourself?  We'd be glad for your assistance!  We can provide you with guidance and code review in return.  If so, tick any of the relevant boxes below:

- [ ] I'd like to try adding this scraper myself
- [x] I'd like guidance to help me develop a scraper
- [x] I'd prefer if the `recipe-scrapers` community try to add this

----

This website doesn't appear to conform to schema.org. I don't know if that's all wordpress sites, or this is a one-off.

I'm not python guru, but can dabble and would love to contribute if possible. Just need a push in the right direction.

**Comment by zemsten** [comment]
Okay so looking into this a bit, the older posts (like the link that I submitted as an example) don't have a defined schema. The new ones do, however. Unfortunately it doesn't get interpretted correctly though. I think this is likely the fault of the author posting content in the "wrong" places as defined by the schema. I haven't been able to get the scraper to pull anything in 100% correctly yet.

**Comment by mkinsella** [comment]
@zemsten were you able to make any additional progress on this? I'd be willing to help out.

**Comment by brianstrauch** [comment]
This looks like a duplicate of https://github.com/hhursev/recipe-scrapers/issues/467? There's a bit more progress over there, so maybe this issue can be closed.

**Comment by jknndy** [comment]
Yep you're right @brianstrauch , thanks for the comment. Closing now, I'll take another look at the open issue sometime in the near future to see if coverage can be added.

**Comment by jknndy** [comment]
Yep you're right @brianstrauch , thanks for the comment. Closing now, I'll take another look at the open issue sometime in the near future to see if coverage can be added.
